src/Symfony/Component/Mime/Test/Constraint/EmailHtmlBodyContains.php

```diff
@@ -44,11 +44,6 @@ protected function matches($message): bool
         return false !== mb_strpos($message->getHtmlBody(), $this->expectedText);
     }
 
-    /**
-     * @param RawMessage $message
-     *
-     * {@inheritdoc}
-     */
```

Comment:
why superfuous here?

Reply:
I thought it was a typo as the annotated param here is `$message`, probably copy pasted from the function above, while the real param is `$email` here but with no indications on it's type it seems. So I decided to simply remove it.

I'll keep the inheritdoc though.